bin/test-dist

```diff
@@ -12,12 +12,12 @@ set -e -u -o pipefail
 : "${TEST_DIRS:=t}"
 
 if [[ -n $CODECOV_ENABLE || -n $COVERALLS_TOKEN || -n $CODE_COVERAGE_REPORT ]]; then
-    HARNESS_PERL_SWITCHES=${HARNESS_PERL_SWITCHES:="-MDevel::Cover=+ignore,^local/|^t/|^xt"}
+    : "${HARNESS_PERL_SWITCHES:="-MDevel::Cover=+ignore,^local/|^t/|^xt"}"
```

Comment:
What does the leading colon do here?

Reply:
It does nothing :)
```
$ help :
:: :
    No effect; the command does nothing.  A zero exit code is returned.
```
The `:=` expansion operator changes a value in place if the variable has no value,
whereas `:-` just returns the given default.

Before you had stuff like:
```
HARNESS_PERL_SWITCHES=${HARNESS_PERL_SWITCHES:="-MDevel::Cover=+ignore,^local/|^t/|^xt"}
```
which would be better written as:
```
HARNESS_PERL_SWITCHES=${HARNESS_PERL_SWITCHES:-"-MDevel::Cover=+ignore,^local/|^t/|^xt"}
```
because `:=` had already changed `HARNESS_PERL_SWITCHES` before re-assigning it to `HARNESS_PERL_SWITCHES`.

But you can't just say:
```
${HARNESS_PERL_SWITCHES:-"-MDevel::Cover=+ignore,^local/|^t/|^xt"}
```
you need to do it as a side effect of some command. 

The Bash idiom for this to use the `:` builtin command for destructive expansion side effects like this.
```
: "${HARNESS_PERL_SWITCHES:="-MDevel::Cover=+ignore,^local/|^t/|^xt"}"
```
Note we now double-quote the expansion because it's a command argument in this context.